### ISSUE TITLE
test-macstadium-macos10.11-x64-2 out of space

### ISSUE THREAD

**Issue by ronag** [nodejs/build#2246]
https://ci.nodejs.org/job/node-test-commit-osx/32898/#showFailuresLink

We might want to label dgram failures on osx1011 as flaky/warning similar to  osx1015.

**Comment by richardlau** [comment]
I've taken https://ci.nodejs.org/computer/test-macstadium-macos10.11-x64-2/ offline for the moment.

**Comment by richardlau** [comment]
https://github.com/nodejs/build/issues/2173 may be related but I don't see any `.fseventsd` folders... @AshCripps could you take a look to see if it's the same issue?

```
bash-3.2$ pwd
/Users/iojs
bash-3.2$ du -hsc $(ls -A)
6.3M    .babel.json
8.0K    .bash_history
4.2G    .ccache
 16K    .config
4.0K    .gitconfig
 94M    .jenkins
2.9M    .node-gyp
4.0K    .node_repl_history
1.3M    .npm
4.0K    .npmrc
4.0K    .rnd
4.0K    .ssh
112K    .standard-cache
384K    .standard-v12-cache
 40K    .standard-v13-cache
168K    .standard-v14-cache
4.0K    .viminfo
  0B    .yarn
4.0K    .yo-rc-global.json
404M    Library
 11G    build
  0B    jenkins.log
 19M    jenkins_diagnostics.txt
108K    jenkins_err.log
  0B    node-icu
652K    node-tmp
852K    slave.jar
4.0K    start.sh
  0B    tmp
 15G    total
bash-3.2$ df -h
Filesystem      Size   Used  Avail Capacity  iused   ifree %iused  Mounted on
/dev/disk0s2    49Gi   43Gi  6.1Gi    88% 11286090 1611206   88%   /
devfs          175Ki  175Ki    0Bi   100%      604       0  100%   /dev
map -hosts       0Bi    0Bi    0Bi   100%        0       0  100%   /net
map auto_home    0Bi    0Bi    0Bi   100%        0       0  100%   /home
bash-3.2$
```

**Comment by gabrielschulhof** [comment]
test-macstadium-macos10.11-x64-2 is still accepting jobs: https://ci.nodejs.org/job/node-test-commit-osx/32915/nodes=osx1011/consoleFull

**Comment by richardlau** [comment]
Someone must have reenabled it :(

I've compared test-macstadium-macos10.11-x64-1 and test-macstadium-macos10.11-x64-2 and disk usage is similar between them except that test-macstadium-macos10.11-x64-2 has a 7Gb workspace for `node-stress-single-test` that I've removed.

https://ci.nodejs.org/job/node-test-commit-osx/32919/nodes=osx1011/ is now running on it. Will keep an eye on it to see if it succeeds.
